docs/tools/vdb_table/data/vectara.json

```diff
@@ -87,23 +87,23 @@
     "comment": ""
   },
   "recsys": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.vectara.com/docs/learn/recommendation-systems/recommender-overview",
```

Comment:
this doc page only mentions doing this via query by text. If that's the only way, then it wouldn't qualify as recsys support. Recsys means users can store arbitrary sorts of embeddings/representations that allow nearest neighbor search in that particular space. For example user-product, user-user, etc.

I believe this is not supported by Vectara. Can you confirm?

Reply:
I see. With that definition, I agree. I've updated the JSON to reflect.